consensus/objs/phlkey.go

```diff
@@ -20,8 +20,8 @@ func (b *PendingHdrLeafKey) UnmarshalBinary(data []byte) error {
 	if len(data) != 34 {
```

Comment:
This check is done in several files. Please write a ticket for yourself to pop that performs the length check against the data using the constants hashlen and the len of the prefix. This will make code more readable and less fragile under modification. This will not block merge of PR.